Sentinel/Classes/Core/Internal/Image+Assets.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Image+Assets.swift
+//  Sentinel
+//
+//  Created by Zvonimir Medak on 10.12.2024..
+//
+
+import SwiftUI
+
+// There's an issue with the generated assets, and local cocoapods which should be resolved in Xcode 16.1
```

Comment:
Open issue for this, so we can track this (Productive or Issues here).

Reply:
I created an issue here so we don't pollute Productive with issues. I think that we have a better reach here with the whole community https://github.com/infinum/ios-sentinel/issues/45